src/platforms/blackpillv2/platform.h

```diff
@@ -29,81 +29,155 @@
 
 #define PLATFORM_HAS_TRACESWO
 #define PLATFORM_IDENT "(BlackPillV2) "
+
+/* Error handling for ALTERNATIVE_PINOUT
+ * If ALTERNATIVE_PINOUT has a value >= 4 (undefined), or <= 0, an error is thrown.
+ */
+#ifdef ALTERNATIVE_PINOUT
+#if ALTERNATIVE_PINOUT <= 0
```

Comment:
please add `|| ALTERNATIVE_PINOUT > 3`, and drop the `#else` + `#warning` branch below - it doesn't make sense to check this condition twice in two different ways.

Reply:
Implemented. Thanks!